extensions/interactions/answer-defs.ts

```diff
@@ -105,6 +105,8 @@ export interface INumberWithUnitsAnswer {
   units: IUnit[];
 }
 
+export type INumericExpressionAnswer = string;
+
```

Comment:
Along with this please also add this def this `IInteractionAnswer`

Reply:
Done.